grails-app/domain/org/transmartproject/db/dataquery/highdim/snp_lz/DeSnpInfo.groovy

```diff
@@ -4,7 +4,7 @@ class DeSnpInfo {
 
     String name
     String chromosome
-    BigDecimal pos
```

Comment:
BigDecimal provides "immutable, arbitrary-precision signed decimal numbers". A SNP position is an integer (Number of size 10 in the database design of DEAPP.DE_SNP_INFO, column CHROM_POS).